cms/envs/test.py

```diff
@@ -136,7 +136,8 @@
     MIGRATION_MODULES = NoOpMigrationModules()
 
 LMS_BASE = "localhost:8000"
-FEATURES['PREVIEW_LMS_BASE'] = "preview"
+FEATURES['PREVIEW_LMS_BASE'] = "preview.localhost"
```

Comment:
I'm not sure why PREVIEW_LMS_BASE doesn't have a port here and other examples do. It seems like it should be consistent.